### PR TITLE
GIX-2085: AccountsPlusPagePo

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

There was a mismatch with the PO used in one test and the actual component being tested.

The spec for the `+page` in the accounts route was using the PO for the `routes/Accounts.svelte`. Which is really a child of that component.

In this PR, I add a new PO `AccountsPlusPagePo`. I added the `PlusPage` in reference to `+page` used in the route. I hope this will help with the misconcenption.

# Changes

* New PO AccountsPlusPagePo
* Use the new PO in `routes/app/accounts/page.spec.ts`

# Tests

Only test changes.

# Todos

- [ ] Add entry to changelog (if necessary).

Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3806** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/3806?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/3806?utm_source=stack-comment).

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/tests/page-objects/AccountsPlusPage.page-object.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { BasePageObject } from "$tests/page-objects/base.page-object";
+import type { PageObjectElement } from "$tests/types/page-object.types";
+import { AccountsPo } from "./Accounts.page-object";
+
+export class AccountsPlusPagePo extends BasePageObject {
+  private static readonly TID = "accounts-plus-page-component";
```
Comment (dskloetd) [inline frontend/src/tests/page-objects/AccountsPlusPage.page-object.ts:6]: This TID doesn't exist anywhere as far as I can tell?

File: frontend/src/tests/page-objects/AccountsPlusPage.page-object.ts
Comment (dskloetd) [inline frontend/src/tests/page-objects/AccountsPlusPage.page-object.ts:1]: Is this for
```
frontend/src//routes/(app)/(u)/(accounts)/+page.svelte
```
or
```
frontend/src//routes/(app)/(u)/(accounts)/accounts/+page.svelte
```
or both and what's the difference between those files?
The contents seems to be identical.

File: frontend/src/tests/page-objects/AccountsPlusPage.page-object.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/tests/page-objects/AccountsPlusPage.page-object.ts:6]: Oops, I forgot to stage those two changes in the PR.

Now

**Comment by lmuntaner** [comment]
@dskloetd ready for another review

**Review by dskloetd — APPROVED** [review]

**Pull request merged**